mopidy_tunein/tunein.py

```diff
@@ -176,10 +179,15 @@ def find_playlist_parser(extension, content_type):
 class TuneIn(object):
     """Wrapper for the TuneIn API."""
 
-    def __init__(self, timeout, session=None):
+    def __init__(self, timeout, filter=None, session=None):
         self._base_uri = 'http://opml.radiotime.com/%s'
         self._session = session or requests.Session()
         self._timeout = timeout / 1000.0
+        self._filter = filter
+        if (filter == translator.get_id_type('s')):
```

Comment:
Sorry to mess you around but I didn't quite mean this, this is rather cryptic. I was thinking:
```
if (filter == translator.TUNEIN_ID_STATION):
...
elif (filter == translator.TUNEIN_ID_PROGRAM):
```
Or even defining a complimentary mapping function in `translator`:
```
def get_guide_id(id_type):
    return {TUNEIN_ID_PROGRAM: 'p',
            TUNEIN_ID_STATION: 's',
            ... etc ... ,
            TUNEIN_ID_STREAM: 'e'}.get(id_type, '')
```
And then something like
```
if filter is not None:
    self._filter = '&filter=' + translator.get_guide_id(filter)
```

Reply:
My first implementation used translator.TUNEIN_ID_STATION, but then i read about the "cyclical dependency".. so i did it cryptic 😆

Reply:
OK, just go back to strings then. I'm not a fan of the cryptic version.

Reply:
we are booth not fans of the cryptic version